### ISSUE TITLE
Benchmark function

### ISSUE THREAD

**Issue by edward-burn** [darwin-eu-dev/PatientProfiles#357]
@catalamarti it would be good to discuss how we could add a benchmark of PatientProfiles. This would help track progress of improvements in the code, but also can be something data partners can run to make sure everything works on their database and is sufficiently performant. 

In IncidencePrevalence we have a dedicated benchmark function. My original preference (and possibly still now) is that this code should live somewhere else (a benchmarking study project?), but would be good to think about so we do something consistent across the different analytic package.

**Comment by edward-burn** [comment]
Some interest related links:
https://duckdb.org/2023/04/14/h2oai.html
https://ursalabs.org/blog/2021-r-benchmarks-part-1/

**Comment by catalamarti** [comment]
Yes, sure, do you think that we can create some random cohorts based in the cdm object and person + observation table and then run the benchmark study?
O do you prefer that we have some "real" cohorts?
@edward-burn

**Comment by edward-burn** [comment]
In IncidencePrevalence we make fake example cohorts to make it quick (as that's not the part we're benchmarking)

**Comment by ablack3** [comment]
tagging myself so I get updates @ablack3 :)
